docs/client/java/partials/java_transport.md

```diff
@@ -0,0 +1,265 @@
+| spark.openlineage.transport.roleArn           | Optional, the roleArn which is allowed to read/write to Kinesis stream                                                                                                                       | some-role-arn    |
+| spark.openlineage.transport.properties.[xxx]  | Optional, the [xxx] is property of [Kinesis allowd properties](https://github.com/awslabs/amazon-kinesis-producer/blob/master/java/amazon-kinesis-producer-sample/default_config.properties) | 1                |
+
+</TabItem>
+<TabItem value="flink" label="Flink Config">
+
+| Parameter                                     | Definition                                                                                                                                                                                   | Example          |
+-----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------
+| openlineage.transport.streamName        | Required, the streamName of the Kinesis Stream                                                                                                                                               | some-stream-name |
+| openlineage.transport.region            | Required, the region of the stream                                                                                                                                                           | us-east-2        |
+| openlineage.transport.roleArn           | Optional, the roleArn which is allowed to read/write to Kinesis stream                                                                                                                       | some-role-arn    |
+| openlineage.transport.properties.[xxx]  | Optional, the [xxx] is property of [Kinesis allowd properties](https://github.com/awslabs/amazon-kinesis-producer/blob/master/java/amazon-kinesis-producer-sample/default_config.properties) | 1                |
+
+
+</TabItem>
+</Tabs>
+
+*Behavior*:
+- Events are serialized to JSON upon the `emit()` call and dispatched to the Kinesis stream.
+- The partition key is generated by combining the job's namespace and name.
+- Two constructors are available: one accepting both `KinesisProducer` and `KinesisConfig` and another solely accepting `KinesisConfig`.
+
+
+### [Console](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/ConsoleTransport.java)
+
+This straightforward transport emits OpenLineage events directly to the console through a logger.
+Be cautious when using the DEBUG log level, as it might result in double-logging due to the `OpenLineageClient` also logging.
+No additional configuration is required.
+
+<Tabs groupId="integrations">
+<TabItem value="yaml" label="OpenLineageYaml Config">
```

Comment:
Is there no spark / flink config here?

Reply:
Yeah. `transport.type` is sufficient and this is documented elsewhere. I thought empty tab would be better than no tab.